routes/utils/generateToken.js

```diff
@@ -0,0 +1,22 @@
+const jwt = require("jsonwebtoken");
+
+module.exports = {
+    generateToken
+}
+
+function generateToken(user) {
+    const payload = {
+      subject: user.id,
+      username: user.username
+    }
+    const options = {
+      expiresIn: '1d',
+    }
+    const result = jwt.sign(
+      payload,
+      'A SECRET CODE HERE',
```

Comment:
It's not great practice to store your secret in the open this way. Have an env variable with your secret and maybe this public secret as a fallback

Reply:
> It's not great practice to store your secret in the open this way. Have an env variable with your secret and maybe this public secret as a fallback

Thank you, Francis. I'll look into it right now.